django/thunderstore/repository/models/uploader_identity.py

```diff
@@ -87,3 +88,35 @@ def can_user_upload(self, user):
             UploaderIdentityMemberRole.owner,
             UploaderIdentityMemberRole.member,
         )
+
+    def can_create_service_account(self, user) -> None:
+        membership = self.members.filter(user=user).first()
+        if not membership:
+            raise ValidationError("Must be a member to create a service account")
+        if membership.role != UploaderIdentityMemberRole.owner:
+            raise ValidationError("Must be an owner to create a service account")
```

Comment:
I don't think a raise parameter is needed here as the only place this will be called is inside the form and the return type should make it clear it raises as opposed to returning a `bool`.

Reply:
In that case I'd rename it to `ensure_can_create_service_account` or something which implies it might throw